docs/02-iaas/deployment-examples/getting-started.mdx

```diff
@@ -0,0 +1,121 @@
+# Getting Started
+
+## Preface
+
+This document describes a possible environment setup for a pre-production or minimal production setup. 
+In general hardware requirements can vary largely from environment to environment and this guide is not 
+a hardware sizing guide nor the best placement solution of services for every setup. This guide intends to 
+provide a starting point for a hardware based deployment of the SCS-IaaS reference implementation based on OSISM.
+
+
+## Node type definitions
+
+### Control Node
+A control node runs all or most of the openstack services, that are responsible for API-services and the corresponding 
+runtimes. These nodes are necessary for any user to interact with the cloud and to keep the cloud in a managed state. 
+However these nodes are usualy **not** running user virtual machines.
+Hence it is advisable to have the control nodes replicated. To have a RAFT-quorum three nodes are a good starting point. 
+
+### Compute Node (HCI/no HCI)
+
+#### No HCI
```

Comment:
```suggestion
#### Not Hyperconverged (no HCI)
```
The abbreviation has not been explained before ...
(OK, ok, maybe we should write "Not Hyperconverged Infra (no HCI)`?)

Reply:
I'll added it